src/main/java/Professor.java

```diff
@@ -0,0 +1,132 @@
+import java.util.ArrayList;
+
+public class Professor extends User {
+
+    private ArrayList<Project> projects;
+    private ArrayList<Project> secondReaderProjects;
+    private ProjectCoordinator projectCoordinator;
+
+
+    public Professor(String firstName, String lastName, String email, ProjectCoordinator projectCoordinator)
+    {
+        super(firstName, lastName, email);
+        this.projects = new ArrayList<>();
+        this.secondReaderProjects = new ArrayList<>();
+        this.projectCoordinator = projectCoordinator;
+    }
+
+    public Professor(String firstName, String lastName, String email)
+    {
+        this(firstName, lastName, email, null);
+    }
+
+
+    /**
+     * Create a new project and add it to the Projects list of the calling Professor and
+     * the Project Coordinator.
+     * @param description Project description
+     * @param restrictions Project restrictions
+     * @param studentCapacity Project capacity (how many students can join)
+     */
+    public void createProject(String description, ArrayList<String> restrictions, int studentCapacity)
+    {
+        Project project = new Project(this, description, restrictions, studentCapacity);
+
+        projectCoordinator.addProjectToList(project);
+        this.addProjectToList(project);
+    }
+
+
+    /**
+     * Delete the project, removing it from the Professor's and the Project Coordinator's
+     * list of proejcts.
+     * @param project The project to delete
+     */
+    public void deleteProject(Project project)
+    {
+        if (projects.contains(project))
+        {
+            projectCoordinator.removeProjectFromList(project); // Should we do this?
+            this.removeProjectFromList(project);
+
+            project.setIsDeleted(true);
```

Comment:
Should we have a state variable in the project that denotes that is deleted or should we just remove all of its references. Discussion issue @justinkrol @DerekStride @noahsegal

Reply:
What is the difference between deleting and archiving from a functional pov if they are both states? 
I don't know if deleting `project` records is a good idea, but why not just soft-delete them by archiving?

Reply:
Removed isDeleted from the Project class. Archive will be our soft-delete.